.github/workflows/container-release.yml

```diff
@@ -0,0 +1,44 @@
+name: Container release
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '5 5 * * 0'
```

Comment:
We don't push tags as much as we merge into master, that's why I did not added to this release workflow.

Reply:
We could add a new action/job to release new tags after merging to master, not sure how you or @ehelms feels about this.

Reply:
I think releasing more often would be a good thing.

Reply:
We could have a job that runs weekly and do auto-tag/release if we have a commit stashed from the week prior.

Reply:
I'd lean more to sumbitting a PR to release. It might be a bit more complicated, but I like the control. If we generalize it, I can see more repos in our org use it